Hazel/src/Hazel/Application.cpp

```diff
@@ -78,4 +79,10 @@ namespace Hazel {
 		return true;
 	}
 
+	bool Application::OnWindowResize(WindowResizeEvent& e)
+	{
+		m_Window->GetContext()->UpdateViewport(e.GetWidth(), e.GetHeight());
+		return true;
```

Comment:
We should not set this as handled, instead, we need to get this event through the layerstack. This way layers can alter there cameras to the new size.